### PR TITLE
:hammer: Update SpawnVehicle

### PR DESCRIPTION
**Pull request by Walker974**
Capture d’écran 2022-05-27 194847

### CODE REVIEW

**Comment by TheSpaceGamerV2** [comment]
It's normal

**Comment by TheSpaceGamerV2** [comment]
linters doesn't support the backticks

File: src/api/vehicles/server/spawnVehicle.lua [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ---@param cb function
 ---@public
 function API_Vehicles:serverSpawn(modelName, coords, cb)
-    local vehicle = Citizen.InvokeNative(`CREATE_AUTOMOBILE`, joaat(modelName), coords)
+    local vehicle = Citizen.InvokeNative('CREATE_AUTOMOBILE', joaat(modelName), coords)
```
Comment (PABLO-1610) [inline src/api/vehicles/server/spawnVehicle.lua:7]:
```suggestion
    local vehicle = CreateVehicle(GetHashKey(modelName), coords)
```

**Review by PABLO-1610 — CHANGES_REQUESTED** [review]

**Comment by Walker974** [comment]
good now ?

**Review by PABLO-1610 — APPROVED** [review]

**Pull request merged**